UI/FeaturesUI.cs

```diff
@@ -224,7 +224,10 @@ private TextBlock GetLabels(dynamic value, int index, int frameRate, bool? varEn
 
             if (double.IsNaN(numValue))
             {
-                textBlock.Value = value.ToString();
+                if (value != null)
```

Comment:
Not sure, but maybe `textBlock.Value = value?.toString();`?

Reply:
when constructed, `textBlock.Value` has a default value of `"Unset"`, so by doing it like this I leave the default value intact and only set it if it has a value.